data-prepper-plugins/opensearch/README.md

```diff
@@ -93,6 +93,8 @@ Default is null.
 
 - `proxy`(optional): A String of the address of a forward HTTP proxy. The format is like "<host-name-or-ip>:\<port\>". Examples: "example.com:8100", "http://example.com:8100", "112.112.112.112:8100". Note: port number cannot be omitted.
 
+- `enable_request_compression` (optional): A boolean that enables or disables request compression when sending requests to OpenSearch. Default is true.
```

Comment:
Would this be a breaking change?

Reply:
It shouldn't be. We unknowingly changed the default from false to true a few months back in #2385 by introducing a new transport client.

The behavior since has been to compress requests. This parameter maintains that behavior by default but also gives an option to disable compression if desired